cmd/argoexec/commands/init.go

```diff
@@ -28,12 +29,12 @@ func loadArtifacts() error {
 	// Download input artifacts
 	err := wfExecutor.StageFiles()
 	if err != nil {
-		wfExecutor.AddError(err)
+		wfExecutor.AddError(errors.Wrap(err, errors.CodeInternal, " Init container failed to stage the files"))
```

Comment:
Any errors returned by our own libraries, should already be wrapped. Thus this wrap is not necessary, and only masks the real source and stacktrace of the original error. This wrap is not necessary.